New logo blogpost (#64)
psf/the-invisibles: content/english/blog/new_logo_2025.md

---
title: "PyPodcats have a new logo"
meta_title: "PyPodcats have a new logo"
description: "Finally after 1 year, we finally have a logo"
date: 2025-06-17T05:00:00Z
categories: ["Blog Post",]
author: "Georgi"
tags: ["python", "community", "women in tech", "women in Python", "women in open source","python underrespresnted groups"]
draft: false
image: "/images/logo-blog.png"
---

## PyPodcats Have a New Logo

> After a year of placeholder art and indecision, we finally have a real logo. And yes—it has a cat. On a mic. With a snake. We promise it makes sense.

Reviewer: ```suggestion
After a year of placeholder art and indecision, the PyPodcats team finally have a real logo. And yes—it has a cat. On a mic. With a snake. We promise it makes sense.
```

> Our original stand-in was a charming sticker from Mariatta’s [“#TypoOfTheDay”](https://mariatta.ca/posts/typo_of_the_day/) series. It wasn’t meant to be a logo, but we were so eager to get *Hidden Figures of Python* out into the world that we just rolled with it. MVP: Minimum Viable Purr.

Reviewer: Move to a new paragraph
```suggestion

Our original stand-in was a charming sticker from Mariatta’s [“#TypoOfTheDay”](https://mariatta.ca/posts/typo_of_the_day/) series. It wasn’t meant to be a logo, but we were so eager to get *Hidden Figures of Python* out into the world that we just rolled with it. MVP: Minimum Viable Purr.
```

> Many drafts, discussions, and digital furballs later, our team chose the new design: a laid-back cat draped over a retro microphone, with a happy snake coiled in—our nod to the Python community that brought us all together.

Reviewer: ```suggestion

After the conclusion of our first season, we started discussing our plans and goals for this year, and among those ideas, one of them was to come up with a "real" logo for our group. Many drafts, discussions, and digital furballs later, our team chose the new design: a laid-back cat draped over a retro microphone, with a happy snake coiled in—our nod to the Python community that brought us all together.
```

> We printed our first batch of stickers just in time for PyCon US this past May. They were glittery. They were gorgeous. They were gone in a flash. Sorry if you missed them—we had a tight budget and only printed a small run using our own pocket money. Blame the import tax on sparkles.

Reviewer: Insert the new logo as an image before this paragraph, and say "Check our new logo. What do you think?"

Author: do you mean the sticker? Because the logo is already on the blogpost top?

Reviewer: Ah yes, if there is picture of the glittery stickers, it would be great. The blog post is looking great! Thank you!

> We printed our first batch of stickers just in time for PyCon US this past May. They were glittery. They were gorgeous. They were gone in a flash. Sorry if you missed them—we had a tight budget and only printed a small run using our own pocket money. Blame the import tax on sparkles.

Reviewer: ```suggestion
We printed our first batch of stickers using this new logo just in time for PyCon US this past May. They were glittery. They were gorgeous. They were gone in a flash. Sorry if you missed them—we had a tight budget and only printed a small run using our own pocket money. Blame the import tax on sparkles.
```

> Since the start of 2025, we’ve been flying solo. The Python Software Foundation changed its grant structure, and we’re no longer under their funding umbrella. So if you’ve laughed, learned, or loved anything from PyPodcats, consider donating on our [👉Open Collective Platform](https://opencollective.com/pypodcats)

Reviewer: ```suggestion
Another update about PyPodcats, in case you missed it, we are no longer eligible for funding from The Python Software Foundation, due to [the recent change](https://pyfound.blogspot.com/2024/12/12psf-grants-program-charter-updates-part-3.html) in its grant structure. If you're curious of what it takes to run a podcast like ours, check our [financial info](/about/financials/) page.

Have you laughed, learned, or loved anything from PyPodcats podcast series? If so, consider donating on our [👉Open Collective Platform](https://opencollective.com/pypodcats). With your support and donation, we will be able to continue producing our podcast (and to print more of those glittery stickers).
```
